### DISCUSSION TITLE
Logs dont stream after enabling remote logs

### DISCUSSION THREAD

**Discussion by Rutvijjs** [apache/airflow#40163] [General]
Hello, 
I recently enabled remote logs into azure blob storage and now my airflow instance in an AKS cluster will not stream logs in the UI as the task is going. It will only show logs after each task is completed. Here are the changes i made in the helm chart: 
```

  - name: "AIRFLOW__LOGGING__REMOTE_LOGGING"
    value: "True"
  - name: "AIRFLOW__LOGGING__REMOTE_LOG_CONN_ID"
    value: "AZ"
  - name: "AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER"
    value: "wasb-airflow-logs" # must be this value
  - name: "AIRFLOW__LOGGING__LOGGING_CONFIG_CLASS"
    value: "log_config.DEFAULT_LOGGING_CONFIG"
```

The log_config file is directly taken from this repo for Airflow 2.6.3. Is there something i'm missing to have the logs be streamed in instead of it waiting for the logs to be added to the remote location? 

Any help/advice is appreciated,
Thanks!

**Comment by AndriiBurenko** [comment]
Hi! I have the same issue for Airflow2 version 2.9.2 taken from this repo too.

**Comment by makrist2** [comment]
@Rutvijjs Hi! Did you manage to solve this problem ? If yes, can you please tell me how exactly you did it?

**Comment by Rutvijjs** [comment]
Hello, Have not been able to solve this yet.
